docs/reference/api/media.md

```diff
@@ -0,0 +1,153 @@
+# Media APIs
+
+
+The `@apostrophecms/attachment` module manages all media file upload. Once uploaded, metadata and organization of image and other files are primarily managed through the respective `@apostrophecms/image` and `@apostrophecms/file` piece types. See the [pieces REST API documentation](./pieces.md) for additional information.
+
+## Endpoints
```

Comment:
Before you jump right into this you should probably mention that this is a two-step dance: create an attachment, then use it as part of the value of a piece.